src/httpd_jsonapi.c

```diff
@@ -83,6 +83,13 @@ safe_json_add_string(json_object *obj, const char *key, const char *value)
     json_object_object_add(obj, key, json_object_new_string(value));
 }
 
+static inline void
```

Comment:
Remove this function, just use json_object_object_add directly like the rest of the code does

Reply:
Ok